tests/test_job.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+"""Unit tests for job endpoints"""
+
+import json
+import pytest
+from src.rev_ai.apiclient import RevAiAPIClient
+from src.rev_ai.models import Job, JobStatus, JobSubmitOptions
+
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+
+JOB_ID = '1'
+MEDIA_URL = "https://example.com/test.mp3"
+CALLBACK_URL = 'https://callback.com/'
+
+
+@pytest.mark.usefixtures("mockclient")
+class TestJobEndpoints():
+    def test_get_job_details_success(self, mockclient):
+        status = 'transcribed'
+        created_on = '2018-05-05T23:23:22.29Z'
```

Comment:
Is there a reason you didn't pull this parameter out into a global constant like JOB_ID? If not please do